jellyfin.tf

```diff
@@ -312,20 +291,41 @@ EOF
  *   if you've provided a domain.
  */
 
-resource "aws_default_vpc" "default" { }
+resource "aws_vpc" "jellyfin_vpc" { 
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+  instance_tenancy     = "default"
+  cidr_block = "10.0.0.0/16"
+}
+
+resource "aws_internet_gateway" "jellyfin_gw" {
+  vpc_id = aws_vpc.jellyfin_vpc.id
+}
+
+resource "aws_default_route_table" "jellyfin_route" {
+  default_route_table_id = aws_vpc.jellyfin_vpc.default_route_table_id
+  route {
+    cidr_block = "0.0.0.0/0"
+    gateway_id = aws_internet_gateway.jellyfin_gw.id
+  }
+}
 
-resource "aws_default_subnet" "default_a" {
+resource "aws_subnet" "jellyfin_a" {
```

Comment:
Definitely wanted to make this change, nice work. Let's add a Name to these via tags, too, though.
Perhaps somethinglike:

```
tags = {
    Name        = "${var.ENVIRONMENT}-jellyfin_a"
}
```

Reply:
Added the tags